astropy/cosmology/io/table.py

```diff
@@ -33,14 +33,17 @@ def from_table(table, index=None, *, move_to_meta=False, cosmology=None):
         (e.g. for ``Cosmology(meta={'key':10}, key=42)``, the ``Cosmology.meta``
         will be ``{'key': 10}``).
 
-    cosmology : str, `~astropy.cosmology.Cosmology` class, or None (optional, keyword-only)
+    cosmology : str or |Cosmology| class or None (optional, keyword-only)
```

Comment:
I don't think the `type` needs to be specified more thoroughly than `str` in this line.
```suggestion
    cosmology : str or type or None (optional, keyword-only)
```